lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -876,12 +884,14 @@ def _log_grpc_payload(self, grpc_request):
 
         if not self._print_grpc_payload:
             return
-        logging.info(log_msg)
+        logging.debug(log_msg)
```

Comment:
I preffer info here as if we're enabling grpc logging I want it to be seen in default configuration which is INFO level

Reply:
Yes. It should be INFO level. Fixed it.